client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/SecurityDocumentationIT.java

```diff
@@ -785,6 +798,39 @@ public void onFailure(Exception e) {
 
             assertTrue(latch.await(30L, TimeUnit.SECONDS));
         }
+
+        {
+            CreateServiceAccountTokenRequest createServiceAccountTokenRequest =
+                new CreateServiceAccountTokenRequest("elastic", "fleet-server", "token1");
+            CreateServiceAccountTokenResponse createServiceAccountTokenResponse =
+                client.security().createServiceAccountToken(createServiceAccountTokenRequest, RequestOptions.DEFAULT);
+
+            AuthenticateResponse response = client.security().authenticate(
+                RequestOptions.DEFAULT.toBuilder().addHeader(
+                    "Authorization", "Bearer " + createServiceAccountTokenResponse.getValue().toString()).build());
+
+            User user = response.getUser();
+            boolean enabled = response.enabled();
+            final String authenticationRealmName = response.getAuthenticationRealm().getName();
+            final String authenticationRealmType = response.getAuthenticationRealm().getType();
+            final String lookupRealmName = response.getLookupRealm().getName();
+            final String lookupRealmType = response.getLookupRealm().getType();
+            final String authenticationType = response.getAuthenticationType();
+            final Map<String, Object> authMetadata = response.getMetadata();
+
+            assertThat(user.getUsername(), is("elastic/fleet-server"));
+            assertThat(user.getRoles(), empty());
+            assertThat(user.getFullName(), equalTo("Service account - elastic/fleet-server"));
+            assertThat(user.getEmail(), nullValue());
+            assertThat(user.getMetadata(), equalTo(Map.of("_elastic_service_account", true)));
+            assertThat(enabled, is(true));
+            assertThat(authenticationRealmName, is("service_account"));
+            assertThat(authenticationRealmType, is("service_account"));
+            assertThat(lookupRealmName, is("service_account"));
+            assertThat(lookupRealmType, is("service_account"));
+            assertThat(authenticationType, is("token"));
+            assertThat(authMetadata, equalTo(Map.of("_token_name", "token1")));
+        }
```

Comment:
I don't understand why this is here, since it's not used in the Documentation.

Reply:
This was to test the authentication response for a service token. But you are right it's not for documentation. I can move it to `SecurityIT`.